tests/scripts/run-xunit-slice.py

```diff
@@ -0,0 +1,82 @@
+import json
+import argparse
+import os
+import shutil
+import subprocess
+import sys
+
+##########################################################################
+# Argument Parser
+##########################################################################
+
+description = 'Tool to generate slice<number>.txt files based off of a slice json file (see perf-slices.json for format)'
+
+parser = argparse.ArgumentParser(description=description)
```

Comment:
I'm wondering if rather than generating txt files, you might be better off just having this script be the driver that invokes the harness on each directory based on the slice configuration.  Then you don't have to lay down files that then get consumed.  You can just do everything in-memory and have a smaller surface area.  Thoughts?